doc/releases/changes.rst

```diff
@@ -19,7 +19,7 @@ Version 5.1.2 (released Oct 3, 2023)
 ----------------------------------------
 
 - Update Vega-Lite from version 5.14.1 to version 5.15.1;
-  see `Vega-Lite Release Notes <https://github.com/vega/vega-lite/releases>`_.
+  see `Vega-Lite 5.15.1 Release Notes <https://github.com/vega/vega-lite/releases/tag/v5.15.1>`_.
```

Comment:
We get some "duplicate key" warnings without this; maybe harmless but might as well avoid them